docs/docs/genai/flavors/langchain/autologging.mdx

```diff
@@ -114,12 +111,7 @@ assert (
 
 # Enable mlflow langchain autologging
 # Note: We only support auto-logging models that do not contain retrievers
```

Comment:
We can remove this comment as well. We should update this page to remove any non-trace related stuff since they're all removed :)

Reply:
Sounds good, but that's beyond the scope of this PR. Follow-up sounds good.